lib/openapi_parser/schemas/response.rb

```diff
@@ -21,9 +21,11 @@ def validate(response_body, response_validate_options)
 
       media_type = select_media_type(response_body.content_type)
       unless media_type
-        raise ::OpenAPIParser::NotExistContentTypeDefinition, object_reference if response_validate_options.strict
+        if response_validate_options.strict && response_body.status_code != 204
```

Comment:
I think a more universal solution would be to not do content-type validation if the response body is empty, regardless of what the status code is, e.g. 
```suggestion
        if response_validate_options.strict && response_body_not_blank
```
This is conceptually similar to the fix used in https://github.com/openapi-library/OpenAPIValidators/pull/59/files